webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/api/DataStructures.scala

```diff
@@ -65,6 +65,9 @@ object ApiToken {
     case tokenRegex(v) => Some(ApiToken(v))
     case _             => None
   }
+
+  // Avoid printing the value in logs
+  override def toString: String = s"[REDACTED ApiToken]"
```

Comment:
It should be in the case class I think, line 58

Reply:
indeed